src/components/NewRequestForm.tsx

```diff
@@ -97,7 +97,9 @@ const NewRequestForm = ({ documentTypes }: Props): JSX.Element => {
             </fieldset>
           </div>
 
-          <Button type="submit">Send request</Button>
+          <Button type="submit" disabled={isSubmitting}>
```

Comment:
i'm not sure how to test that - isSubmitting comes straight from formik

Reply:
do you mean the whole form?

Reply:
Ideally the whole form, but seeing as we don't have tests for it we should at least try to cover the changes we make. 

In this case we could either mock out formik (this would probably be a PITA) or we include it in the test of the component—so here we would click the button to submit the form and expect to see it disabled.

Reply:
i've added an extra test case for double-submission

Reply:
(or will once we solve the typescript error)

Reply:
done

Reply:
🙌